app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,41 @@
+class VotesController < ApplicationController
+  def index
+    @votes = Vote.all
+  end
+
+  def new
+    user = @logged_in_user
+    work = Work.find_by(id: params[:work_id])
```

Comment:
Again, you've implemented the `index` and `new` actions for the `VotesController`, but they're not used by your site, and you don't have view templates for them.